libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/StreamingExtensions/StreamingExtensionRequestHandler.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Protocol;
+using Microsoft.Bot.Protocol.WebSockets;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Integration.AspNet.Core.StreamingExtensions
+{
+    internal class StreamingExtensionRequestHandler : RequestHandler
```

Comment:
I'm not understanding the role this class is intended to play. Could you add signifigant comments explaining what this class does, and how it fits into the larger picture?